javascript/packages/cli/src/actions/convert.ts

```diff
@@ -0,0 +1,130 @@
+import type {
+  NodeConfig,
+  PL_ConfigType,
+  ParachainConfig,
+  PolkadotLaunchConfig,
+} from "@zombienet/orchestrator";
+import { decorators, getFilePathNameExt } from "@zombienet/utils";
+import fs from "fs";
+import path from "path";
+import { DEFAULT_BALANCE } from "../constants";
+
+export async function convert(param: string) {
+  try {
+    const filePath = param;
+
+    if (!filePath) {
+      throw Error("Path of configuration file was not provided");
+    }
+
+    // Read through the JSON and write to stream sample
+    await convertInput(filePath);
+  } catch (err) {
+    console.log(
+      `\n ${decorators.red("Error: ")} \t ${decorators.bright(err)}\n`,
+    );
+  }
+}
+
+// Convert functions
+// Read the input file
+async function readInputFile(
+  ext: string,
+  fPath: string,
+): Promise<PL_ConfigType> {
+  let json: object;
+  if (ext === "json" || ext === "js") {
+    json =
+      ext === "json"
+        ? JSON.parse(fs.readFileSync(`${fPath}`, "utf8"))
+        : await import(path.resolve(fPath));
+  } else {
+    throw Error("No valid extension was found.");
+  }
+  return json;
+}
+
+async function convertInput(filePath: string) {
+  const { fullPath, fileName, extension } = getFilePathNameExt(filePath);
+
+  const convertedJson = await readInputFile(extension, filePath);
+
+  const { relaychain, parachains, simpleParachains, hrmpChannels, types } =
+    convertedJson;
+
+  let jsonOutput: PolkadotLaunchConfig;
+  const nodes: NodeConfig[] = [];
+  const paras: ParachainConfig[] = [];
+  let collators: NodeConfig[] = [];
+
+  const DEFAULT_NODE_VALUES = {
+    validator: true,
+    invulnerable: true,
+    balance: DEFAULT_BALANCE,
+  };
+
+  parachains &&
```

Comment:
```suggestion```

Not needed, we have default in the destructuring at the start of the function